README.md

```diff
@@ -92,14 +104,23 @@ Usage inject:
 
 Flags:
   -p, --config-path string   path to the local configuration overriding config-url (default ".golic.yaml")
-  -u, --config-url string    default config URL (default "https://raw.githubusercontent.com/AbsaOSS/golic/main/.golic.yaml")
```

Comment:
is it deprecated?

Reply:
Yes, at the first stages we read master configuration from github repo, user could customize master config by changing config URL. Currently, master configuration is embedded resource and is compiled into golic binary. Locl `.golic.yaml` is local configuration overriding master configuration. For further details see: https://github.com/AbsaOSS/golic#golicyaml